setup.cfg

```diff
@@ -20,7 +20,7 @@ include_package_data = True
 
 [options.extras_require]
 sentry =
-    sentry_sdk~=0.19.0
+    sentry_sdk~=1.10
```

Comment:
major version bump and no changes needed? amazing 🤩

Reply:
those sentry folks know what they're doing